doc/releases/release-notes-2.3.rst

```diff
@@ -34,8 +34,18 @@ The following CVEs are addressed by this release:
   into a fixed-size array.
 * CVE-2020-10059: UpdateHub Module Explicitly Disables TLS
   Verification
-* CVE-2020-10062: Under embargo until 2020/05/25
+* CVE-2020-10061: Improper handling of the full-buffer case in the
+  Zephyr Bluetooth implementation can result in memory corruption.
+* CVE-2020-10062: Packet length decoding error in MQTT
 * CVE-2020-10063: Under embargo until 2020/05/25
```

Comment:
We're past the 25th of May, if this is what this date is @d3zd3z ?

Reply:
I think I need to just remove the line entirely. It turns out to not qualify for a CVE.